source/index-management.rst

```diff
@@ -185,6 +204,44 @@ Standard API
     listIndexes(): Cursor;
   }
 
+  interface CreateIndexOptions {
+    /**
+     * Specifies how many data-bearing members of a replica set, including the primary, must
+     * complete the index builds successfully before the primary marks the indexes as ready.
+     *
+     * This option accepts the same values for the "w" field in a write concern plus "all",
+     * which indicates all voting data-bearing nodes.
+     *
+     * @note This option is only available in MongoDB 4.4+.
+     *
+     * Drivers MUST NOT throw an error if this option is specified when creating an index on
+     * an earlier version of the server. Instead, drivers MUST rely on the server for returning
+     * an appropriate error.
+     *
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     */
+    commitQuorum: Optional<Int32 | String>;
+
+    /**
+     * The maximum amount of time to allow the index build to take before returning an error.
+     *
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     */
+    maxTimeMS: Optional<Int64>;
+  }
+
+  interface CreateIndexesOptions {
```

Comment:
If there's no distinction at all, lets only document one type `CreateIndexOptions` and use it in both places, rather than describe combining them as an acceptable deviation.